students/morgan/dnd_project/bin/generator.py

```diff
@@ -0,0 +1,65 @@
+from encounter_builder.interface import ui
+from encounter_builder.data_models import party_data, monster_data
+from encounter_builder.util import number_crunchers
+# import setup
+# import encounter_builder.resource_files
+
+
+
+if __name__ == "__main__":
+
+
+
+    # xp_budget_file = setup.package_data['encounter_budget_by_level']
+    xp_budget_file = "encounter_builder/resource_files/xp_budget_by_level.csv"
```

Comment:
you probably want to structure the data a bit to make it more robust and flexible:

`res_dir = Path(encounter_builder/resource_files)`

then: 

`xp_budget_file =res_dir / "xp_budget_by_level.csv"`

etc....